x-pack/rest-resources-zip/build.gradle

```diff
@@ -19,6 +19,7 @@ configurations {
 dependencies {
   apis project(path: ':rest-api-spec', configuration: 'restSpecs')
   freeTests project(path: ':rest-api-spec', configuration: 'restTests')
+  freeTests project(path: ':modules:aggregations', configuration: 'restTests')
```

Comment:
The clients were going to lose tests every time we port things from rest-api-spec into the aggregations module without this.